.github/workflows/ci.yml

```diff
@@ -74,3 +112,5 @@ jobs:
         run: |
           java -version
           mvn -B test
+          aws s3 cp s3://aws-crt-test-stuff/ci/${{ env.BUILDER_VERSION }}/linux-container-ci.sh ./linux-container-ci.sh && chmod a+x ./linux-container-ci.sh
```

Comment:
This is messed up a little.  Installing java within the runner isn't going to affect the build inside the container, so it's not really doing anything.  Also we're building/testing on both the runner (ubuntu) and the container (al2).  I would just revert this particular change.

Reply:
In that case, is there a way for use test different Java version?